benchmarks/dynamo/huggingface.py

```diff
@@ -437,6 +437,7 @@ def forward_pass(self, mod, inputs, collect_outputs=True):
     def forward_and_backward_pass(self, mod, inputs, collect_outputs=True):
         cloned_inputs = clone_inputs(inputs)
```

Comment:
curious, why do we clone inputs here?

Reply:
this is to keep the original inputs unmodified, when the model mutates the inputs.